bin/makeDocs

```diff
@@ -127,8 +127,12 @@ class Doxyfile(object):
                     for path, dirs, files in os.walk(entry):
                         for f in files:
                             fullName = os.path.join(path, f)
-
-                            contents = "".join(open(fullName).readlines())
+                            contents = ""
+                            with open(fullName) as fh:
+                                try:
+                                    contents = "".join(fh.readlines())
+                                except UnicodeDecodeError:
```

Comment:
Yes, the story is that this `open` can open any files it finds in the doc directory and those include PNG files which really upset Unicode decoding... I guess I'll add a comment about that.